src/utils/stringToBoolean.js

```diff
@@ -0,0 +1,12 @@
+export const stringToBoolean = string => {
+  switch (string.toLowerCase().trim()) {
+    case 'true':
+      return true
+    case 'false':
+    case '0':
+    case null:
```

Comment:
What about `case 'undefined'`, `case 'NaN'`, `case ''`?
They are also considered “falsy” values in JS, in addition to `false`, `0`, & `null`:
```js
Boolean(undefined) === false
Boolean(NaN) === false
Boolean('') === false
```